src/Blocks/Bookmark.php

```diff
@@ -34,7 +35,7 @@ class Bookmark implements BlockInterface
     private function __construct(Block $block, string $url, array $caption)
     {
         if (!$block->isBookmark()) {
-            throw new \Exception("Block must be of type " . self::TYPE);
+            throw new BlockTypeException(self::TYPE);
```

Comment:
I solved the unkilled mutation here by throwing a named exception and setting the message in there. The change is repeated for most of the `Blocks` and their tests.